src/controllers/artists.js

```diff
@@ -0,0 +1,55 @@
+const Artist = require('../models/artist');
+const { find } = require('mongoose');
+
+exports.create = (req, res) => {
+  const artist = new Artist({
+    name: req.body.name,
+    genre: req.body.genre,
+  });
+
+  artist.save().then(() => {
+    res.status(201).json(artist);
+  });
+};
+
+exports.list = (req, res) => {
+  Artist.find({}, (err, artists) => {
+    res.status(200).send(artists);
+  });
+};
+
+exports.find = (req, res) => {
+  Artist.findById(req.params.artistId, (err, Aid) => {
+    if (!Aid) {
+      res.status(404).json({ error: 'The artist could not be found.' });
+    } else {
+      res.status(200).json(Aid);
+    }
+  });
+};
+
+exports.patch = (req, res) => {
+  Artist.findById(req.params.artistId, (err, artist) => {
+    if (!artist) {
+      res.status(404).json({ error: 'The artist could not be found.' });
+    } else {
```

Comment:
You are repeating the same response a few times. Could be worth putting it into a helper function. 
Maybe something like:
```
const _handleArtistNotFound = (res) => {
    res.status(404).json({ error: 'The artist could not be found.' });
}
```

then in your controller:
```
if (!artist) {
    _handleArtistNotFound(res);
}
```

Reply:
You could even create a bit of middleware to handle all of your cases where an `_id` is not found.